R/init.R

```diff
@@ -100,3 +104,14 @@ build_site_meta <- function(pkg = ".") {
   write_yaml(meta, path_meta)
   invisible()
 }
+
+is_non_pkgdown_site <- function(dst_path) {
+  if (!dir_exists(dst_path)) {
+    return(FALSE)
+  }
+
+  top_level <- dir_ls(dst_path)
+  top_level <- top_level[!path_file(top_level) %in% c("CNAME", "dev")]
+
+  length(top_level) > 1 && !"pkgdown.yml" %in% path_file(top_level)
```

Comment:
I'm confused about this test. What do we test here for?

Reply:
If there are top level files in the docs directory and pkgdown.yml is not in them. i.e. a site in `docs/` not built by pkgdown.

Reply:
`> 1` ?